wdl/tasks/Pacasus.wdl

```diff
@@ -0,0 +1,180 @@
+version 1.0
+
+import "Utils.wdl" as Utils
+import "Structs.wdl"
+
+workflow Process {
+    input {
+        File reads
+        Int chunk_size_mb
+    }
+
+    call Utils.ConvertReads {
+        input:
+            reads = reads,
+            output_format = "fasta"
+    }
+
+    call SplitReadsByLength {
+        input:
+            reads_fasta = ConvertReads.converted_reads,
+            length = 55000
+    }
+
+    call SplitReads {
+        input:
+            reads_fasta = SplitReadsByLength.shorter_reads,
+            chunk_size_mb = chunk_size_mb
+    }
+
+    scatter (split_read_fasta in SplitReads.split_reads_fasta) {
+        call RemovePalindromes {
+            input:
+                read_fasta = split_read_fasta
+        }
+    }
+
+    call MergeFasta {
+        input:
+            processed_fastas = flatten([RemovePalindromes.processed_fasta, [SplitReadsByLength.longer_reads]])
+    }
+
+    output {
+        File processed_fasta = MergeFasta.processed_fasta
+    }
+}
+
+task SplitReadsByLength {
+    input {
+        File reads_fasta
+        Int length
+    }
+
+    Int disk_size = 3 * ceil(size(reads_fasta, "GB"))
+
+    command <<<
+        set -euxo pipefail
+
+        seqkit seq ~{reads_fasta} --max-len $(expr ~{length} + 1) > shorter_reads.fasta
+        seqkit seq ~{reads_fasta} --min-len ~{length} > longer_reads.fasta
+    >>>
+
+    output {
+        File shorter_reads = "shorter_reads.fasta"
+        File longer_reads = "longer_reads.fasta"
+    }
+
+    runtime {
```

Comment:
Coupled with the change above, the runtime section should have the RuntimeAttr preamble, like so:

```
    #########################
    RuntimeAttr default_attr = object {
        cpu_cores:          1,
        mem_gb:             1,
        disk_gb:            50,
        boot_disk_gb:       10,
        preemptible_tries:  3,
        max_retries:        1,
        docker:             "us.gcr.io/broad-dsp-lrma/lr-pb:0.1.8"
    }
    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
    runtime {
        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
    }
```